src/content/components/ProfileCallTreeSettings.js

```diff
@@ -27,17 +27,21 @@ class ProfileCallTreeSettings extends Component {
   }
 
   render() {
-    const { jsOnly, invertCallstack, searchString } = this.props;
+    const { implementationFilter, invertCallstack, searchString } = this.props;
     return (
       <div className='profileCallTreeSettings'>
         <ul className='profileCallTreeSettingsList'>
           <li className='profileCallTreeSettingsListItem'>
             <label className='profileCallTreeSettingsLabel'>
-              <input type='checkbox'
+              Filter:
+              <select
```

Comment:
ah yes, in normalize.css v5: https://github.com/necolas/normalize.css/blob/5.0.0/normalize.css#L258